manifests/host-pod.yaml.j2

```diff
@@ -14,8 +14,7 @@ spec:
   containers:
   - name: {{ pod_name }}
     image: {{ test_image }}
-    command:
-      - "{{ command }}"
+    command: {{ command }}
```

Comment:
Shouldnt the ENTRYPOINT be a single string? Any additional args would be provided as an {{ arg }} IIUC

Reply:
As far as Kubernetes is concerned, the "command" (akin to the ENTRYPOINT) and the "args" (akin to the CMD) are both lists.

In practice, we only pass the command name there, so our list will only have one entry. But it's a list still.
So for us, it would also work to keep it as it was. I find it a bit nicer, to have this however a list in the template.